opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_view_instrument_match.py

```diff
@@ -0,0 +1,110 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from logging import getLogger
+from threading import Lock
+from typing import Iterable, Set
+
+from opentelemetry.sdk._metrics.aggregation import (
+    _convert_aggregation_temporality,
+)
+from opentelemetry.sdk._metrics.measurement import Measurement
+from opentelemetry.sdk._metrics.point import AggregationTemporality, Metric
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
+
+_logger = getLogger(__name__)
+
+
+class _ViewInstrumentMatch:
+    def __init__(
+        self,
+        name: str,
```

Comment:
Why not just pass in the view here to get all of these properties? They are well defined there and then if more fields are added to the view, you don't need to update this constructor

Reply:
These attributes don't necessarily come from the view, they may come from the instrument, like [this](https://github.com/open-telemetry/opentelemetry-python/pull/2301/files#diff-693118fd4a59966d0e9d2499bb9704ce0af972e3390e712b3a7488e0848009bbR77).